cloudevents/spec.md

```diff
@@ -436,15 +436,25 @@ on the definition of OPTIONAL.
 - Constraints:
   - OPTIONAL
   - If present, MUST be a non-empty string
-- Example:
+- Examples:
   - A subscriber might register interest for when new blobs are created inside a
     blob-storage container. In this case, the event `source` identifies the
     subscription scope (storage container), the `type` identifies the "blob
     created" event, and the `id` uniquely identifies the event instance to
     distinguish separate occurrences of a same-named blob having been created;
     the name of the newly created blob is carried in `subject`:
-    - `source`: `https://example.com/storage/tenant/container`
-    - `subject`: `mynewfile.jpg`
+      - `source`: `https://example.com/storage/tenant/container`
+      - `subject`: `mynewfile.jpg`
+  - A subscriber might register interest for when new updates are made to a client
+    in an eCommerce system. In this case, the event `source` identifies the
+    subscription scope (CRM part of an eCommerce system), the `type` identifies
+    the "client updated" event, and the `id` uniquely identifies the event
+    instance to distinguish separate occurrences of a same client being
+    updated multiple times; the `subject` uniquely identifies the client within
+    the scope of the `source` by including a "partner id" and "client id"
+    (which is unique within the scope of the "partner id") separated by a colon:
+      - `source`: `https://example.com/eCommerce/crm`
+      - `subject`: `5:100`
```

Comment:
@evarga, @jskeet  had a minor suggested edit on today's call:
```
`subject`: `partnerid/5/clientid/100`
```
so it's a bit more descriptive. Would you be ok with this?